src/library_fs_shared.js

```diff
@@ -45,6 +45,7 @@ addToLibrary({
   $FS_createPreloadedFile__deps: [
     '$asyncLoad',
     '$PATH_FS',
+    '$FS_createDataFile',
```

Comment:
Should this be updated in the function body below as well?

Reply:
Good catch, this happened to work because all users of this do actually include `FS` anyhow, I guess. But it's better to be precise.